src/main/java/gwt/material/design/amcharts/client/bullet/Bullet.java

```diff
@@ -56,4 +57,9 @@
     @JsMethod
     public native void deepInvalidate();
 
+    @JsOverlay
+    public final void propertyFields_disabled(Object propertyField, String state) { /*-{
+        propertyField.disabled = state;
```

Comment:
Is there a way to JSInterop this?

Reply:
This was intended for setting propertyField that is a JS Object type that is a collection of key/value pairs, in JS you will write:
`bullet.propertyFields.disabled = "disabled";`
I didn't expose propertyFields directly to Java because there is not a corresponding java type, so with this overlay is possible to set the "disabled" property, in JAVA, by calling:
`bullet.propertyFields_disabled(bullet.propertyFields, "disabled");`
This in not very nice, because you should write an overlay method for every property but I don't have a better idea. If you have suggestions they are very welcome and I will try to implement.